src/subcommand/server.rs

```diff
@@ -2091,6 +2115,33 @@ mod tests {
     );
   }
 
+  #[test]
+  fn sat_from_n_endpoint() {
+    let server = TestServer::new_with_regtest_with_index_sats();
+    server.mine_blocks(1);
+
+    let txid = server.bitcoin_rpc_server.broadcast_tx(TransactionTemplate {
+      inputs: &[(1, 0, 0, inscription("text/foo", "hello").to_witness())],
+      ..Default::default()
+    });
+
+    server.mine_blocks(1);
+
+    let inscription_id = InscriptionId { txid, index: 0 };
+
+    server.assert_response_regex(
+      "/r/sat/5000000000/0",
+      StatusCode::OK,
+      format!(r#".*\{{"id":"{}"\}}.*"#, inscription_id),
+    );
+
+    server.assert_response_regex(
+      "/r/sat/5000000000/-1",
```

Comment:
We shoudl think about what happens if you pass in an index that is out of bounds. Should we return a 404 or a json with a descriptive error?